fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/catalog/FlinkCatalogITCase.java

```diff
@@ -190,35 +192,40 @@ void testPartitionedTable() throws Exception {
         ObjectPath objectPath = new ObjectPath(DEFAULT_DB, "test_partitioned_table");
 
         // 1. first create.
-        tEnv.executeSql("create table test_partitioned_table (a int, b string) partitioned by (b)");
+        tEnv.executeSql(
+                "create table test_partitioned_table (a int, b string, dt string) partitioned by (b,dt)");
         Schema.Builder schemaBuilder = Schema.newBuilder();
-        schemaBuilder.column("a", DataTypes.INT()).column("b", DataTypes.STRING());
+        schemaBuilder
+                .column("a", DataTypes.INT())
+                .column("b", DataTypes.STRING())
+                .column("dt", DataTypes.STRING());
         Schema expectedSchema = schemaBuilder.build();
         CatalogTable table = (CatalogTable) catalog.getTable(objectPath);
         assertThat(table.getUnresolvedSchema()).isEqualTo(expectedSchema);
         List<String> partitionKeys = table.getPartitionKeys();
-        assertThat(partitionKeys).isEqualTo(Collections.singletonList("b"));
+        assertThat(partitionKeys).isEqualTo(Arrays.asList("b", "dt"));
 
         // 2. add partitions.
-        tEnv.executeSql("alter table test_partitioned_table add partition (b = 1)");
-        tEnv.executeSql("alter table test_partitioned_table add partition (b = 2)");
-        tEnv.executeSql("alter table test_partitioned_table add partition (b = 3)");
-        List<String> expectedShowPartitionsResult = Arrays.asList("+I[b=1]", "+I[b=2]", "+I[b=3]");
+        tEnv.executeSql("alter table test_partitioned_table add partition (b = 1,dt = 1)");
+        tEnv.executeSql("alter table test_partitioned_table add partition (b = 2,dt = 1)");
+        tEnv.executeSql("alter table test_partitioned_table add partition (b = 3,dt = 1)");
+        List<String> expectedShowPartitionsResult =
+                Arrays.asList("+I[dt=1/b=1]", "+I[dt=1/b=2]", "+I[dt=1/b=3]");
```

Comment:
This looks like something is wrong. Why the `dt` is displayed before `b` partition value? Theoretically, the partition values should be in the order of partition field definition `b, dt`.